### DISCUSSION TITLE
Example of a Functioning Custom Tool

### DISCUSSION THREAD

**Discussion by mrgizmo212** [mckaywrigley/chatbot-ui#1277] [Help]
Can anyone provide an example of a functioning custom tool?

**Comment by mckaywrigley** [comment]
Closing since we tackled this earlier :)

**Reply by mrgizmo212** [reply]
Cool thank you again.

**Reply by adrianpunkt** [reply]
Where exactly was this tackled before?! Would be useful to see some example instead of reverse engineering to understand how they work and how to set up tools
